config-templates/google/gcp-compute-persistent-disk-csi-driver/1.8.0/deployment.yaml

```diff
@@ -0,0 +1,848 @@
+            env:
+            - name: GOOGLE_APPLICATION_CREDENTIALS
+              value: /etc/cloud-sa/cloud-sa.json
+            image: k8s.gcr.io/cloud-provider-gcp/gcp-compute-persistent-disk-csi-driver:v1.7.2
+            name: gce-pd-driver
+            volumeMounts:
+            - mountPath: /csi
+              name: socket-dir
+            - mountPath: /etc/cloud-sa
+              name: cloud-sa-volume
+              readOnly: true
+          hostNetwork: true
+          nodeSelector:
+            kubernetes.io/os: linux
+          priorityClassName: csi-gce-pd-controller
+          serviceAccountName: csi-gce-pd-controller-sa
+          volumes:
+          - emptyDir: {}
+            name: socket-dir
+          - name: cloud-sa-volume
+            secret:
+              secretName: cloud-sa
+  - apiVersion: apps/v1
+    kind: DaemonSet
+    metadata:
+      name: csi-gce-pd-node
+      namespace: gce-pd-csi-driver
+      labels:
+        app.kubernetes.io/name: gcp-compute-persistent-disk-csi-driver
+        app.kubernetes.io/version: 1.8.0
+    spec:
+      selector:
+        matchLabels:
+          app: gcp-compute-persistent-disk-csi-driver
+          app.kubernetes.io/name: gcp-compute-persistent-disk-csi-driver
+          app.kubernetes.io/version: 1.8.0
+      template:
+        metadata:
+          labels:
+            app: gcp-compute-persistent-disk-csi-driver
+            app.kubernetes.io/name: gcp-compute-persistent-disk-csi-driver
+            app.kubernetes.io/version: 1.8.0
+          annotations:
+            autoUpdate: "AUTO-UPDATE-ENABLED"
+        spec:
+          containers:
+          - args:
+            - --v=5
+            - --csi-address=/csi/csi.sock
+            - --kubelet-registration-path=/var/lib/kubelet/plugins/pd.csi.storage.gke.io/csi.sock
+            env:
+            - name: KUBE_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+            image: k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.5.0
+            livenessProbe:
+              exec:
+                command:
+                - /csi-node-driver-registrar
+                - --kubelet-registration-path=/var/data/kubelet/plugins/pd.csi.storage.gke.io/csi.sock
+                - --mode=kubelet-registration-probe
+              initialDelaySeconds: 3
+            name: csi-driver-registrar
+            volumeMounts:
+            - mountPath: /csi
+              name: plugin-dir
+            - mountPath: /registration
+              name: registration-dir
+          - args:
+            - --v=5
+            - --endpoint=unix:/csi/csi.sock
+            - --run-controller-service=false
+            image: k8s.gcr.io/cloud-provider-gcp/gcp-compute-persistent-disk-csi-driver:v1.7.2
+            name: gce-pd-driver
+            securityContext:
+              privileged: true
+            volumeMounts:
+            - mountPath: /var/lib/kubelet
```

Comment:
change this to /var/data/

Reply:
Changed here: https://github.com/IBM/ibm-satellite-storage/pull/221/commits/0f41c691643168e11046fa58c0916fd334695a3d